packages/deeplinks/observability/locators/logs_explorer.ts

```diff
@@ -27,6 +27,22 @@ export type ListFilterControl = {
 
 export const LOGS_EXPLORER_LOCATOR_ID = 'LOGS_EXPLORER_LOCATOR';
 
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+export type DocumentFieldGridColumnOptions = {
+  type: 'document-field';
+  field: string;
+  width?: number;
+};
+
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+export type SmartFieldGridColumnOptions = {
+  type: 'smart-field';
+  name: 'content' | 'resource';
+  width?: number;
+};
```

Comment:
Any reason why you have the same type defined at `x-pack/plugins/observability_solution/logs_explorer/common/display_options/types.ts`? 

I was a bit confused reviewing the PR wondering where the `fallbackFields` prop was coming from.

Reply:
So we have 2 challenges here
1. Why can't we define the type in one place and use them everywhere ? - We currently don't have a common place to define the types which can be imported both the `deep-link` package and in `logExplorer` plugin, Hence we had to repeat them. Also they both are not linked, though they look same, hence had to duplicate them.
2. `fallbackFields` is an implementation detail for `LogExplorer`. We don't want users of `locators` to know about an underlying implementation detail on how columns would fallback. Hence it was removed from `locators` type and thus also the reason, why we sort of duplicated the logic in locators.

Reply:
Ah, I didn't see it was on different solutions. It's ok then.